x-pack/plugins/security/server/audit/audit_events.ts

```diff
@@ -131,6 +131,29 @@ export function userLoginEvent({
   };
 }
 
+export interface UserLogoutParams {
+  username?: string;
```

Comment:
__question:__ do we intentionally log `undefined` for the audit events without `username`? It looks a bit weird that we record `{... user: { name: undefined } }`, is there any reason why we prefer it to something like this `{... user: undefined }`?

Reply:
No reason in particular, happy to change it